src/ddo/NvmConfig.ts

```diff
@@ -44,6 +44,13 @@ export interface NvmConfig {
     versions: NvmConfigVersions[]
 }
 
+export enum ImmutableBackendsSupported {
+  Filecoin = 'filecoin',
+  IPFS = 'ipfs',
+  Arweave = 'arweave',
+  Other = 'other'
```

Comment:
Does it make sense to have both of these here if they are not supported yet?

Reply:
changed